tests/fsharp/SDKTests/tests/DefaultFSharpCore - IsNotPreviewSdkEnablePreviewCheck.proj

```diff
@@ -0,0 +1,21 @@
+<Project ToolsVersion="4.0" DefaultTargets="Test" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+
+  <!-- Verifies that in an RTM sdk with the previewcheck disabled it selects the already shipped fsharp.core package -->
+
+  <PropertyGroup>
+    <_NETCoreSdkIsPreview>false</_NETCoreSdkIsPreview>
+  </PropertyGroup>
+
+  <Import Project="Test.props" />
+
+  <PropertyGroup>
+    <ExpectsValueTuple>false</ExpectsValueTuple>
+    <DisableImplicitSystemValueTupleReference>true</DisableImplicitSystemValueTupleReference>
+    <DisableFSharpCorePreviewCheck>false</DisableFSharpCorePreviewCheck>
+    <ExpectsFSharpCore>true</ExpectsFSharpCore>
+    <ExpectedFSharpCorePackageVersion>$(FSCorePackageVersion)</ExpectedFSharpCorePackageVersion>
```

Comment:
The test comment states that it should select the already shipped version, isn't that named $(FSharpCoreShippedPackageVersion)?